test/index.spec.js

```diff
@@ -78,3 +79,32 @@ test('Quadbin geometryToCells', async t => {
   }
   t.end();
 });
+
+test('cellToBoundary works with quadbins', t => {
+  const result = cellToBoundary(ANY_QUADBIN);
+
+  t.equal(result.type, 'Polygon', 'Should return a Polygon');
+  t.ok(Array.isArray(result.coordinates), 'Coordinates should be an array');
+  t.equal(result.coordinates.length, 1, 'Should have one boundary array');
+
+  t.ok(result.coordinates[0].length === 5, 'Boundary should have 5 points');
+
+  t.end();
+});
+
+test('cellToBoundary works with Quadbins near the antimeridian', t => {
+  for (const quadbin of [
+    BigInt(536903670), // Longitude near +180°
+    BigInt(536870921) // Longitude near -180°
+  ]) {
+    const result = cellToBoundary(quadbin);
+
+    t.equal(result.type, 'Polygon', 'Should return a Polygon');
+    t.ok(
+      result.coordinates[0][0][0] > 170 || result.coordinates[0][0][0] < -170,
```

Comment:
Better to define a list of `TEST_CASES` with quadbins and expected bounds and then just iterate over them